graphql/server.go

```diff
@@ -157,9 +159,12 @@ func (c *conn) handleSubscribe(id string, subscribe *subscribeMessage) error {
 		ctx = c.makeCtx(ctx)
 
 		start := time.Now()
+		span, ctx := opentracing.StartSpanFromContext(ctx, "thunder.subscription")
+		ext.SamplingPriority.Set(span, 1)
```

Comment:
I think that will require some refactoring in parser.go. Would be good to do anyway, because it will make for better logging.